csharp/ql/src/Stubs/make_stubs_nuget.py

```diff
@@ -211,16 +211,14 @@ def run_cmd(cmd, msg="Failed to run command"):
             copiedFiles.add(pathInfo)
             shutil.copy2(pathInfos[pathInfo], frameworkDir)
 
-exitCode = 0
 for pathInfo in pathInfos:
     if pathInfo not in copiedFiles:
         print('Not copied to nuget or framework folder: ' + pathInfo)
         othersDir = os.path.join(stubsDir, 'others')
         if not os.path.exists(othersDir):
             os.makedirs(othersDir)
         shutil.copy2(pathInfos[pathInfo], othersDir)
-        exitCode = 1
 
 print("\n --> Generated structured stub files: " + stubsDir)
 
-exit(exitCode)
+exit(0)
```

Comment:
Why was this changed?

Reply:
Because
```
python3 ql/src/Stubs/make_stubs_nuget.py webapp Swashbuckle.AspNetCore.Swagger latest "$STUBS_PATH"
```
fails to make one stub to a package, but it is not really an issue.